.github/workflows/nightly-release.yml

```diff
@@ -81,12 +76,10 @@ jobs:
           path: dist/
           merge-multiple: true
           pattern: wheel-*
-      - run: ls -lah dist/
       - run: |
-          version="${{ needs.build.outputs.version }}"
-          tag_version=$(echo $version | sed 's/\.cu[0-9.]*torch[0-9.]*$//')
+          ls -lah dist/
+          tag_version=$(ls dist | sed 's/flashinfer-\(.*g[0-9a-f]\{7\}\).*/\1/' | uniq)
```

Comment:
This is incorrect because the download will include multiple versions of whls.

Reply:
It doesn't matter. All files share the same base version and git hash and we have `uniq` here.

Reply:
Using uniq here will also result in multiple git hashes here.

Reply:
Currently there is only one unique hash in artifacts like https://github.com/flashinfer-ai/flashinfer-nightly/actions/runs/12345123776/job/34449087048#step:3:8. In which case will we have multiple hashes?

Reply:
> 0s
Run ls -lah dist/
total [1](https://github.com/flashinfer-ai/flashinfer-nightly/actions/runs/12353721475/job/34474267132#step:3:1).6G
drwxrwxr-x 2 flashinfer flashinfer 4.0K Dec 1[6](https://github.com/flashinfer-ai/flashinfer-nightly/actions/runs/12353721475/job/34474267132#step:3:7) 13:46 .
drwxrwxr-x 6 flashinfer flashinfer 4.0K Dec 16 13:38 ..
-rw-rw-r-- 1 flashinfer flashinfer 399M Dec 16 13:46 flashinfer-0.0+g08405a5.d20241216.cu121torch2.4-cp38-abi3-linux_x86_64.whl
-rw-rw-r-- 1 flashinfer flashinfer 400M Dec 16 13:46 flashinfer-0.0+g08405a5.d20241216.cu124torch2.4-cp38-abi3-linux_x86_64.whl
-rw-rw-r-- 1 flashinfer flashinfer 2.0M Dec 16 13:46 flashinfer-0.0+g08405a5.d20241216.tar.gz
-rw-rw-r-- 1 flashinfer flashinfer 388M Dec 16 13:12 flashinfer-0.1.6+d9d8eb1.cu12.1torch2.4-cp38-abi3-linux_x86_64.whl
-rw-rw-r-- 1 flashinfer flashinfer 1.[7](https://github.com/flashinfer-ai/flashinfer-nightly/actions/runs/12353721475/job/34474267132#step:3:8)M Dec 16 13:12 flashinfer-0.1.6+d9d8eb1.cu12.1torch2.4.tar.gz
-rw-rw-r-- 1 flashinfer flashinfer 3[8](https://github.com/flashinfer-ai/flashinfer-nightly/actions/runs/12353721475/job/34474267132#step:3:9)9M Dec 16 13:12 flashinfer-0.1.6+d9d8eb1.cu12.4torch2.4-cp38-abi3-linux_x86_64.whl
-rw-rw-r-- 1 flashinfer flashinfer 1.7M Dec 16 13:12 flashinfer-0.1.6+d[9](https://github.com/flashinfer-ai/flashinfer-nightly/actions/runs/12353721475/job/34474267132#step:3:10)d8eb1.cu12.4torch2.4.tar.gz

Reply:
https://github.com/flashinfer-ai/flashinfer-nightly/actions/runs/12353721475/job/34474267132

Reply:
> https://github.com/flashinfer-ai/flashinfer-nightly/actions/runs/12353721475/job/34474267132

See https://github.com/flashinfer-ai/flashinfer-nightly/actions/runs/12353721475/job/34474267132#step:4:3.

It is using the old action files...

Reply:
That's why I added `find dist -type f ! -name "*${tag_version}*" -delete` as a workaround.

Reply:
It will retrieve multiple files, including the latest ones. If CI continues with your current changes without resolving this issue, problems will arise.

Reply:
> It will retrieve multiple files, including the latest ones. If CI continues with your current changes without resolving this issue, problems will arise.

Got it, I will fix it.